includes/classes/class-rtcamp-google-embeds.php

```diff
@@ -442,6 +453,36 @@ public function get_thumb_preview( \WP_REST_Request $request ) {
 		$data['preview_url'] = $this->get_thumbnail_url( $file_id );
 		return new \WP_REST_Response( $data, 200 );
 	}
+
+	/**
+	 * Save thumbnail of a doc.
+	 *
+	 * @param string $file_id File ID.
+	 * @param mixed  $contents Contents of file.
+	 *
+	 * @return void
+	 */
+	public function save_thumbnail( $file_id, $contents ) {
+		$upload_dir = wp_upload_dir();
+		$basedir    = '';
+
+		if ( ! empty( $upload_dir['basedir'] ) ) {
```

Comment:
No need to check basedir here 😅 , it'll always return the same array.

Reply:
Hmm. https://github.com/rtCamp/wp-google-drive/pull/6#discussion_r419899851